include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/CaptureResult.kt

```diff
@@ -20,11 +21,11 @@ sealed interface CaptureResult {
     override val compareFile: File?
       get() = null
 
-    override fun toJson(): JSONObject {
-      val json = JSONObject()
-      json.put("type", "recorded")
-      json.put("golden_file_path", goldenFile.absolutePath)
-      json.put("timestamp", timestampNs)
+    override fun toJson(): JsonObject {
+      val json = JsonObject()
```

Comment:
This is better than what we have now, but I though we could use auto mapping👀
https://github.com/google/gson/blob/main/UserGuide.md#object-examples